spec/controllers/payments_controller_spec.rb

```diff
@@ -70,18 +75,4 @@
     end
   end
 
-  describe 'GET #success' do
-
-    it 'calls User#grant_membership if membership payment' do
-      application
-
-      expect(payment.user.member).to be false
-      expect(payment.user.membership_number).to be_nil
-
-      get :success, params: { id: payment, user_id: payment.user.id }
-
-      expect(payment.reload.user.member).to be true
-      expect(payment.user.membership_number).not_to be_nil
-    end
-  end
```

Comment:
I think this test in useful to retain as it confirms that the controller action - upon payment success - including triggering the membership status update mechanism.

The test description should be changed to something like: 'triggers membership update mechanism'.

What do you think?

Reply:
Yes --  we could have the test be something like:
```
it 'calls payment.successfully_completed ' do
  < expect that payment receives :successfully_completed message>
end
```
Doing more than that seems to me that we're then testing Observable.